src/mcp/tools/auth/set_auth_claims.ts

```diff
@@ -0,0 +1,31 @@
+import { z } from "zod";
+import { tool } from "../../tool.js";
+import { mcpError, toContent } from "../../util.js";
+import { setCustomClaim } from "../../../gcp/auth.js";
+
+export const set_auth_claims = tool(
+  {
+    name: "set_auth_claims",
+    description: "Sets a list of specific claims for a user.",
+    inputSchema: z.object({
+      uid: z.string().describe("the UID or localId of the user to update"),
+      claim: z.string().describe("the key value int he custom claim to update"),
+      value: z
+        .union([z.string(), z.boolean(), z.number()])
+        .describe("the value of the custom claim"),
+    }),
+    annotations: {
+      title: "Set a custom claim on a specific user.",
+      destructiveHint: true,
```

Comment:
```suggestion
```

I don't think this is destructive per se (unless we want to mark *all* write ops as destructive). I think since there's a `readonlyHint` and a `destructiveHint` that leaves a middleground for "neither hint" which means it's not readonly but it's also not destructive.

Reply:
Ahh fair. I can change this